### ISSUE TITLE
undefined method `sanitize'

### ISSUE THREAD

**Issue by cohki0305** [panorama-ed/order_as_specified#26]
I got an error in developing following readme.
Please tell me how to resolve the problem.

### code

```ruby
class ModelObject
  extend OrderAsSpecified
end
```

### error

> pry(main)) > ModelObject.all.order_as_specified(....)
> NoMethodError: undefined method `sanitize' for #<Class:0x007fbed55982d8>
Did you mean?  sanitize_sql

## Env

rails 5.1.1
ruby 2.4.1
PostgreSQL

**Comment by cohki0305** [comment]
Error gone if I change following code.

```diff
- "#{table}.#{attribute}=#{sanitize(value)}"
+"#{table}.#{attribute}=#{sanitize_sql(value)}"
```

https://github.com/panorama-ed/order_as_specified/blob/21a812dd7827b6b113692c20079eaad91c5f2fb0/lib/order_as_specified.rb#L26

**Comment by cohki0305** [comment]
I can order by string following code.
```
"#{table}.#{attribute}='#{sanitize_sql(value)}'"
```

**Comment by fergyfresh** [comment]
What versions of ActiveRecord should I test with this? I have a couple hours tonight and this seems like a fun thing to dig into and clean up for you guys. Is it only `4.0.1` or do we want more?

**Comment by JacobEvelyn** [comment]
@fergyfresh I saw your comment that it looks like you deleted. Are you still interested in looking into this? I'm happy to help!

**Comment by fergyfresh** [comment]
Interested in helping. Didn't want to waste your time as I found what I was looking for in the test failures under the `AR_VERSION` env.

**Comment by JacobEvelyn** [comment]
No worries! In case it wasn't clear we're hoping for this to work in all versions of ActiveRecord from 4.0.1 onward (including 5.x).

**Comment by JacobEvelyn** [comment]
Closing this, as it was fixed in #28. The fixed version is v1.3. Thanks for all the help in identifying this issue!
